Python/ceval.c

```diff
@@ -107,7 +107,7 @@ static void
 dump_stack(_PyInterpreterFrame *frame, PyObject **stack_pointer)
```

Comment:
Are the changes in this function intentional, or left over from debugging?

Reply:
Left over from debugging, I needed them as `dump_stack` was too slow for a realistic number (many millions) of instructions for finding bugs.

It might be worth making this change, but that's for another PR.